src/connector/src/source/kafka/client_context.rs

```diff
@@ -132,9 +158,11 @@ impl KafkaContextCommon {
                         )
                         .await
                     })
-                });
-                handle.join().unwrap()??
+                })
+                .unwrap()
+                .unwrap()
```

Comment:
shall we avoid possible panic? just inspect errors and print them

Reply:
I think the only way is to return the errors. We need to either return some value or an error, according to the function signature.

Reply:
I dont think we need handle it delicately, at least we need to know what the error is.